backend/tet/events/tests/test_services.py

```diff
@@ -11,40 +11,51 @@
     ADD_EVENT_PAYLOAD,
     ADD_EVENT_RESPONSE,
     EVENT_RESPONSE_NO_CUSTOM_DATA,
-    EVENT_RESPONSE_NO_USER,
     EVENT_RESPONSE_OTHERUSER,
+    EVENT_RESPONSE_TEST_COMPANY,
     EVENT_RESPONSE_TESTUSER_EMAIL,
     EVENT_RESPONSE_TESTUSER_OID,
     SAMPLE_EVENTS,
 )
 
 LOGGER = logging.getLogger(__name__)
+TEST_COMPANY_BUSINESS_ID = (
+    "654321-5"  # Needs to match linked_events_responses.EVENT_RESPONSE_TEST_COMPANY
```

Comment:
Maybe use this value in EVENT_RESPONSE_TEST_COMPANY or get this value from EVENT_RESPONSE_TEST_COMPANY so that it doesn't have to be duplicated?